lib/ansible/modules/cloud/scaleway/scaleway_user_data.py

```diff
@@ -102,7 +108,20 @@ def get_user_data(compute_api, server_id, key):
         msg = 'Error during user_data patching: %s %s' % (response.status_code, response.body)
         compute_api.module.fail_json(msg=msg)
 
-    return response.json
+    return response.body
+
+
+def lookup_user_data(compute_api, server_id):
+    user_data_list = compute_api.get(path="servers/%s/user_data" % server_id)
+    if not user_data_list.ok:
+        msg = 'Error during user_data fetching: %s %s' % (user_data_list.status_code, user_data_list.body)
```

Comment:
`user_data_list.body` is `None` when an error occurs (same as [here](https://github.com/ansible/ansible/pull/45121#discussion_r217891190)):
```
"msg": "Error during user_data fetching: 400 None"
```